python/ray/experimental/serve/examples/utils.py

```diff
@@ -0,0 +1,10 @@
+"""
+Utility functions for serve examples.
+"""
+
+from ray.experimental.serve.utils import pformat_color_json
+
+
+def pprint_color_json(d):
```

Comment:
This function is pretty useless would be better to just call `print(pformat_color_json(d))` directly